index.html

```diff
@@ -30,8 +30,7 @@
     </header>
     <main>
         <section class="headline" id="intro">
-            <h1>I'm Khalil</h1>
-            <h1>Glad to see you!</h1>
+            <h1>I'm Khalil <br> Glad to see you!</h1>
```

Comment:
Changed the h1 tags to ensure there is only one h1 tag in the documents.
update heading h1